src/data/schema.ts

```diff
@@ -16,6 +16,7 @@ const utilityFunctionSchema = z.union([z.string(), utilityFunctionNewStructure])
 
 export namespace base {
   export const measureSingle = z.object({
+    name: z.string(),
```

Comment:
next time this should be in a separate PR, because otherwise this won't get merged until the rest of the PR gets merged

Reply:
Gotcha! I forgot i have already addressed this in the refactor branch... Next time will make another PR for this type of changes.